Resources/Prototypes/DeltaV/Entities/Mobs/Player/human.yml

```diff
@@ -15,3 +15,17 @@
     - type: AutoImplant
       implants:
       - DeathAcidifierImplant
+
+- type: entity
+  noSpawn: true
+  parent: MobHumanSyndicateAgent
```

Comment:
Using the death squad names is probably fine, but this could also be improved with the addition of flavor text by adding a description:
Given that this entity isn't affiliated with Gorlex, it would probably be more appropriate to have it use the human name lists.